src/Utilities/copy.jl

```diff
@@ -9,9 +9,39 @@
 
 include("copy/index_map.jl")
 
-_sort_priority(::Any) = 2
-_sort_priority(::MOI.UserDefinedFunction) = 0
-_sort_priority(::MOI.ObjectiveSense) = 1
```

Comment:
So in retrospect, all this PR really needed was `_sort_priority(::Any) = 3` and `_sort_priority(::MOI.ObjectiveFunction) = 2`.

So we want to make this all public?

Reply:
I would do this simple fix, and reopen this PR if this keeps coming back and someone actually needs to extend this kind of methods.